### ISSUE TITLE
Audit host processes which can't be tracked by host apparmor profile due to apparmor limitation

### ISSUE THREAD

**Issue by oneiro-naut** [kubearmor/KubeArmor#247]
There is a known limitation of apparmor that it can't confine the processes that were executed before some profile was enforced.
Following set of steps can be used to reproduce this behaviour:
-  Log into host shell(shell1) 
-  Log into another instance of host shell(shell2) and run kubearmor in it. Run it with hostpolicy enabled.
```
vagrant@kubearmor-dev:~/KubeArmor/KubeArmor$ make && make run-with-host
```
- Apply a hostpolicy from shell1. In this example we apply a policy which is supposed to block the `screen` command on host.
```
vagrant@kubearmor-dev:~$ kubectl apply -f KubeArmor/examples/host-security-policies/hsp-kubearmor-dev-proc-path-block.yaml 
kubearmorhostpolicy.security.kubearmor.com/hsp-kubearmor-dev-proc-path-block created
``` 
- Now test the policy in the same shell(shell1). Its not blocked and it is a known limitation of apparmor.
```
vagrant@kubearmor-dev:~$ screen -h
Use: screen [-opts] [cmd [args]]
 or: screen -r [host.tty]
```
- Exit shell1. And log in again to launch a new shell.
```
vagrant@kubearmor-dev:~$ logout
Connection to 127.0.0.1 closed.
```
- Test the hostpolicy in the new shell. Now its blocked.
```
vagrant@kubearmor-dev:~$ screen -h
-bash: /usr/bin/screen: Permission denied
```
As a result of this limitation we can't enforce host apparmor profile on processes that were executed before kubearmor started. But we can surely audit these events.
For example here in the first case even when screen command wasn't being blocked we still receive its `passed` log which are currently treated as `HostLog` and dropped. Instead we can convert these HostLogs to `Audit (Block)` log or `Audit (Allow)` which can then be matched against the hostpolicy applied.

**Comment by nam-jaehyun** [comment]
@oneiro-naut does it work with such a simple change?

**Comment by oneiro-naut** [comment]
@nam-jaehyun It does because policymatcher already showed these logs. The only thing was the action and result shown in the matchedhostlogs was not as we expect. For example sometimes(for example if a shell was already there when we executed kubearmor from other shell) we got Block but the result was passed. Also the audit mode condition wasn't there for the hostpolicy block of code(which we can test on minikube which uses audit mode). So I took that condition from container block of code here https://github.com/kubearmor/KubeArmor/blob/61d12a69a09c02a59ee65df21feab51eb13f0759/KubeArmor/feeder/policyMatcher.go#L914
